samples/FsAsync.js

```diff
@@ -0,0 +1,44 @@
+var fs = require('fs');
+
+console.log('File System sample');
+
+fs.open('testfile.txt', 'w+', function(err, fd) {
+    console.log('opened file error=' + err);
+    fs.stat('testfile.txt', function(err, stats) {
+        if (stats.isFile()) {
+            console.log('fd is file');
+        } else if (stats.isDirectory()) {
+            console.log('fd is directory');
+        } else {
+            console.log('fd has unknown type');
+        }
+    });
+    var wbuf = new Buffer('write some bytes');
+    fs.write(fd, wbuf, 0, wbuf.length, 0, function(err, written, buffer) {
+        console.log('wrote ' + written + ' bytes');
+        var rbuf = new Buffer(16);
+        fs.read(fd, rbuf, 0, 16, 0, function(err, num, buf) {
+            console.log('read from file; error=' + err);
+            console.log('buffer=' + buf.toString('ascii'));
+        });
+    });
+});
+
+fs.open('file.txt', 'w', function(err, fd) {
```

Comment:
This is weird because you're not saving the returned file descriptor number. Should probably do that and at least close it again later.